packages/gasket-plugin-nextjs/lib/build.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const { promisify } = require('util');
+const exec = promisify(require('child_process').exec);
+const builder = require('next/dist/build').default;
+
+const { createConfig } = require('./config');
+
+/**
+ * Checks if Next.js Telemetry is enabled. If so, it disables it.
+ *
+ * https://nextjs.org/telemetry#how-do-i-opt-out
+ *
+ * @param {object} logger gasket logger
+ */
+async function checkTelemetryStatus(logger) {
```

Comment:
naming nit:  The name implies that it's just checking, but it does in-fact disable it. maybe `ensure` instead of `check`?

Reply:
Great feedback. Will update.